src/platforms/flutter/upload-debug.mdx

```diff
@@ -18,32 +18,53 @@ For Flutter Desktop (Windows/macOS/Linux) `split-debug-info` and `obfuscate` fla
 
 Errors raised from the native layer in Flutter apps require certain debug information files to be uploaded. For example, an Android app can use `proguard` for minification and obfuscation. And when using NDK, dwarf debug files need to be uploaded. Flutter Web requires sourcemaps and iOS apps also require dwarf debug information files.
 
-## Sentry Dart Plugin
+## Automatically Upload Debug Symbols
 
-[Try out the Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin), which uploads debug symbols automatically for you, or follow the manual steps below.
+The most straight forward way is to [use the Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin), which uploads debug symbols automatically for you.
 
-## Uploading for iOS and macOS
+### :clipboard: Install
+
+In your `pubspec.yaml`, add `sentry_dart_plugin` as a new dev dependency.
+
+```yaml
+dev_dependencies:
+  sentry_dart_plugin: ^1.0.0
+```
+
+### Build
+
+The `flutter build apk`, `flutter build ios` (or _macos_) or `flutter build web` is required before executing the `sentry_dart_plugin` plugin, because the build spits out the debug symbols and source maps.
+
+### Run
```

Comment:
We have to document the configuration block before the `Run`.

https://github.com/getsentry/sentry-dart-plugin#configuration-optional
Including all the options.

We can make a table similar to https://docs.sentry.io/product/cli/dif/#upload-options